### DISCUSSION TITLE
How to delete or edit a system once added?

### DISCUSSION THREAD

**Discussion by rmleonard** [henrygd/beszel#289] [Support] [answered]
I added a system to monitor and my fat fingers transposed digits in the IP address, How does one edit a system once added?

**Comment by henrygd** [comment, marked as answer]
There is a deletion option in the table on the home page. Click the three dots on the right side of the table row.

If you want to edit the IP address instead of deleting/recreating, go to the `systems` collection table in the PocketBase admin UI (`/_/#/collections?collectionId=2hz5ncl8tizk5nx`). Click the system and it will let you update the host value.

**Comment by matthew-larner** [comment]
This is really confusing if you're on the table view. You need to switch to the grid view first to see the 3 dots. I suggest that the 3 dot icon should be added to the table view.

**Reply by svenvg93** [reply]
I can see the 3 dots on the table view as well next to the bell icon.
<img width="451" height="327" alt="Scherm­afbeelding 2025-07-23 om 08 29 54" src="https://github.com/user-attachments/assets/2065266e-41c3-438d-b36c-cccef8012063" />

**Reply by henrygd** [reply]
@matthew-larner Can you provide a screenshot please? Sounds like you may need to scroll horizontally a bit.

**Reply by pCoder3000** [reply]
you have to scroll i was also not able to find it xDD
